README.md

```diff
@@ -44,3 +51,93 @@ the webhook to send all necessary event types to the endpoint where the bot
+
+    I include whitespace:
+
+```
+
+The details on each option are as follows:
+
+- `categories`: This contains a nested mapping of github labels to changelog
+  sections. If specified at release time when qiskit-bot generates the changelog
+  it will look at each merged PR in the release and if any have any matching
+  labels that commit summary message will be put under the corresponding
+  sections in the changelog used for the release page. If a value for any label
+  is set to `null` this means that this label is counted as matching but will
+  not be included in the generated changelog. By default any labels outside this
+  set will not be included in the changelog but when the
+  `tools/generate_changelog.py` script is run it flags any merged PRs for a pending
+  release that don't have a matching label. By setting one or more labels to `null`
+  these PRs will not show up in that script.
+
+  If this field is not specified the following default values are used:
+  ```yaml
+  "Changelog: Deprecation": Deprecated
+  "Changelog: New Feature": Added
+  "Changelog: API Change": Changed
+  "Changelog: Removal": Removed
+  "Changelog: Bugfix": Fixed
+  "Changelog: None": null
+  ```
+
+- `notifications`: This contains a mapping of path regexes to a list of usernames
+  to notify if an opened PR touches files that match a particular regex (as found
+  by Python's stdlib `re.search()` function). For example if you set the path regex
+  to `".*"` this would match everything, but using a regex gives you control over
+  exactly how and what matches a particular group. If a path matches the listed
+  usernames will be listed in the notification comment left by the bot on a new PR
+  being opened. This is an additive, so if there is more than 1 match the users
+  from all the matches will be listed in that comment. If this is not specified (and
+  `always_notify` is not set) then no comment will be left by the bot when new PRs
+  are opened.
+- `always_notify`: If this is specified a notification/comment is always left on PR
+  opening even if there are no matching notification paths. In the case of no
+  matching paths just the notification prelude is used.
+- `notification_prelude`: If this is specified the text used for this field will
+  be used as the beginning of every notification comment. If this is not specified
+  the following prelude is used:
+
+  ```
+  Thank you for opening a new pull request.
+
+  Before your PR can be merged it will first need to run and pass continuous
+  integration tests and be also be reviewed. Sometimes the review process can
+  be slow, so please be patient.
+
+  While you're waiting on CI and for review please feel free to review other open
+  PRs. While only a subset of people are authorized to approve pull requests for
+  merging everyone is encouraged to review open pull requests. Doing reviews
+  helps reduce the burden on the core team and helps make the project's code
+  better for everyone.
```

Comment:
This might want updating with other typo fixes.

Reply:
Yeah, I'll update it here and in the tests after the last commit